transmartApp/grails-app/init/BootStrap.groovy

```diff
@@ -87,13 +89,24 @@ class BootStrap {
         }
 
         try { // find location of R scripts of RModules
-            def f = ctx.getResource('WEB-INF/Rscripts').getFile()
-            if (!f || !f.isDirectory()) {
-                f = ctx.getResource('classpath:Rscripts').getFile()
+
+            def rmoduleScriptDir
+
+            if (Environment.current == Environment.PRODUCTION) {
+                rmoduleScriptDir = ctx.getResource('WEB-INF/Rscripts').getFile()
+            } else {
+                rmoduleScriptDir = new File('../Rmodules/src/main/resources/Rscripts')
+            }
+
+            if (!rmoduleScriptDir || !rmoduleScriptDir.isDirectory()) {
+                throw new RuntimeException('Could not determine proper for ' +
+                        'Rscript directory')
             }
-            c.RModules.pluginScriptDirectory = f.absolutePath
+
+            c.RModules.pluginScriptDirectory = rmoduleScriptDir.absolutePath
```

Comment:
Does it still work when you do
 
    grails prod run-app
and 
 
    java -Dgrails.env=prod -jar build/libs/mywar-0.1.war 
?